source/dagger/gameplay/ping_pong/ping_pong_main.h

```diff
@@ -2,12 +2,19 @@
 
 #include "core/core.h"
 #include "core/system.h"
+#include "core/game.h"
+#include "core/engine.h"
 
 using namespace dagger;
 
-namespace pingPong
```

Comment:
I think we should leave namespace here, it'll be easier with code complete

Reply:
Do you think `snake_case` looks better for namespaces? I kinda do, maybe I'd change that if it's all the same to you.